kickoff/dockerflow.py

```diff
@@ -0,0 +1,45 @@
+from os import path
+
+from flask import Response, jsonify
+# Can't concatenate the next line with the following one. exc is not defined
```

Comment:
There's something fishy I don't understand about the import here. Is is something known with flask?